src/from_filesystem.cc

```diff
@@ -0,0 +1,52 @@
+// Copyright 2018 Ryan Dahl <ry@tinyclouds.org>
+// All rights reserved. MIT License.
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <string>
+
+#include "third_party/v8/include/v8.h"
+#include "third_party/v8/src/base/logging.h"
+
+#include "deno.h"
+#include "file_util.h"
+#include "internal.h"
+
+namespace deno {
+
+Deno* NewFromFileSystem(void* data, deno_recv_cb cb) {
+  printf("Reading javascript runtime bundle from " BUNDLE_LOCATION "\n");
+
+  std::string js_source;
+  CHECK(deno::ReadFileToString(BUNDLE_LOCATION, &js_source));
+
+  Deno* d = new Deno;
+  d->currentArgs = nullptr;
+  d->cb = cb;
+  d->data = data;
+  v8::Isolate::CreateParams params;
+  params.array_buffer_allocator =
+      v8::ArrayBuffer::Allocator::NewDefaultAllocator();
+  params.external_references = external_references;
```

Comment:
This line is unnecessary (has to do with snapshots. sorry for not catching it earlier)